share/eduvpn/builder/mainwindow.ui

```diff
@@ -342,20 +342,6 @@
                                     <property name="position">1</property>
                                   </packing>
                                 </child>
-                                <child>
```

Comment:
maybe just comment it out for now and add a comment with the link to the issue describing that we still need to add this